specificatie/naam.yaml

```diff
@@ -0,0 +1,173 @@
+openapi: 3.0.3
+info:
+  title: Naam definities
+  version: 2.0.0
+  contact: {}
+paths: {}
+components:
+  schemas:
+    Aanhef:
+      type: string
+      description: |
+        Kun je gebruiken als aanhef in een brief gericht aan persoon.
+      example: "Hoogwelgeboren heer"
+    Aanschrijfwijze:
+      type: object
+      description: |
+        Samengestelde naam en aanspreekvorm die je kunt gebruiken in de communicatie met de persoon.
+      properties:
+        naam:
+          type: string
+          example: "H.W. baron van den Aedel"
+        aanspreekvorm:
+          description: |
+            Deze regel moet als aparte regel boven de samengestelde naam worden geplaatst. Komt alleen voor bij personen met een adellijke titel of predicaat.
+          type: string
+          example: "De hoogwelgeboren heer"
+    GebruikInLopendeTekst:
+      type: string
+      description: |
+        Naam van persoon die je kunt gebruiken als je in lopende tekst (bijvoorbeeld in een brief) aan persoon refereert.
+      example: "baron Van den Aedel"
+    Geslachtsnaam:
+      type: string
+      maxLength: 200
+      description: |
+        De achternaam van een persoon.
+      example: "Vries"
+    VolledigeNaam:
+      type: string
+      description: |
+        Combinatie van voornamen, voorvoegsels en geslachtsnaam, zonder predikaat, adellijke titel of toevoeging van een eventuele partnernaam.
+      example: "Pieter Jan de Vries"
+    Voorletters: 
+      type: string
+      description: |
+        De voorletters van de persoon, afgeleid van de voornamen.
+      example: "P.J."
+    Voornamen:
+      type: string
+      maxLength: 200
+      description: |
+        De verzameling namen voor de geslachtsnaam, gescheiden door spaties.
+    Voorvoegsel:
+      type: string
+      maxLength: 10
+      example: "de"
+    WaardetabelMetSoort:
+      allOf:
+        - $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+        - properties:
+            soort:
+              type: string
+          example:
+            value:
+              code: "JV"
+              omschrijving: "jonkvrouw"
+              soort: "predikaat"
+    GbaNaamBasis:
+      type: object
+      properties:
+        voornamen:
+          $ref: '#/components/schemas/Voornamen'
+        voorvoegsel:
+          $ref: '#/components/schemas/Voorvoegsel'
+        geslachtsnaam:
+          $ref: '#/components/schemas/Geslachtsnaam'
+        adellijkeTitelPredikaat:
+          $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
```

Comment:
Dit moet ook een verwijzing naar WaardetabelMetSoort worden.

Reply:
Waarom? GBA heeft toch alleen maar een code in de registratie? @fsamwel, @CathyDingemanse  Als de RVIG implementatie soort en omschrijving bijlevert wordt het dan we/niet een bewerking?

Reply:
Dat is geen bewerking. Dus mag door RvIG geleverd worden.

In het LO GBA wordt bij adellijke titel predikaat beschreven dat dit een waarde uit de landelijke tabel is. Het is geen informatievraag/bewerking om naast de code ook de omschrijving uit de tabel te halen, dan is het ook geen informatievraag/bewerking om daar soort uit dezelfde tabel aan toe te voegen.

Reply:
> Dat is geen bewerking. Dus mag door RvIG geleverd worden.
> 
> In het LO GBA wordt bij adellijke titel predikaat beschreven dat dit een waarde uit de landelijke tabel is. Het is geen informatievraag/bewerking om naast de code ook de omschrijving uit de tabel te halen, dan is het ook geen informatievraag/bewerking om daar soort uit dezelfde tabel aan toe te voegen.

Ok. Dus hier wel een $ref naar AdellijkeTitelPredikaatType?

Reply:
> Dus hier wel een $ref naar AdellijkeTitelPredikaatType?

graag

Reply:
> Dat is geen bewerking. Dus mag door RvIG geleverd worden.
> 
> In het LO GBA wordt bij adellijke titel predikaat beschreven dat dit een waarde uit de landelijke tabel is. Het is geen informatievraag/bewerking om naast de code ook de omschrijving uit de tabel te halen, dan is het ook geen informatievraag/bewerking om daar soort uit dezelfde tabel aan toe te voegen.

Geldt dit ook voor alle andere waardetabellen? Wordt de omschrijving geleverd bij de code? Zou wel fijn zijn om dit consistent door te trekken

Reply:
ja